src/sonic-host-services/scripts/sonic-host-server

```diff
@@ -13,9 +13,17 @@ import dbus.mainloop.glib
 
 from gi.repository import GObject
 
-def register_modules():
+def find_module_path():
+    """Find path for host_moduels"""
+    try:
+        from host_modules import host_service
+        return os.path.dirname(host_service.__file__)
+    except Exception as e:
```

Comment:
>Exception [](http://example.com/codeflow?start=11&length=9)

Use more specific exception type, otherwise, it may hide bug, such as syntax error.